### PR TITLE
Feature/update header design

### PR DESCRIPTION
**Pull request by noahub**
This PR adds new styling to the sticky header and sidebar. These changes include:
- Converts sidebar content to a more semantic nav tag
- Updates the sidebar background and converts links list to an unordered list
- Adds styling and improves responsiveness of the sticky header content
- Updates logo

Stickybar UX for this PR
Screen Shot 2020-03-20 at 9 56 43 PM

Sidebar UX for this PR
Screen Shot 2020-03-20 at 9 31 38 PM

### CODE REVIEW

File: client/src/components/NavLink.js [hunk 1]
```diff
@@ -1,84 +1,96 @@
...
+              className={styles.NavLink}
+              class="ui visible right demo vertical sidebar labeled icon menu"
+            >
+              <nav>
+                <ul>
+                  <li>
+                    <a
+                      target="_blank"
+                      href="https://about.covid19webapp.com/#lp-pom-block-118"
+                    >
```
Comment (aviskarkc10) [inline client/src/components/NavLink.js]:
```suggestion
                    <a
                      target="_blank"
                      rel="noopener noreferrer"
                      href="https://about.covid19webapp.com/#lp-pom-block-118"
                    >
```

when using `target="_blank"` we should always use `rel="noopener noreferrer"`.

for more information:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

let's add this in other anchor tags too

**Comment by aviskarkc10** [comment]
@noahub there are also conflicts

**Comment by noahub** [comment]
Thanks for the feedback @aviskarkc10! Will resolve the conflicts and push up your suggestions :)

**Comment by noahub** [comment]
Ok, I've removed the [wip] tag, added your suggestions, and resolved the conflicts. Updated the screenshots in the description too.

**Comment by aviskarkc10** [comment]
LGTM :+1:

**Pull request merged**